common/lib/protocol-definitions/src/tokens.ts

```diff
@@ -66,23 +66,52 @@ export interface ITokenClaims {
     jti?: string;
 }
 
+/**
+ * TODO
+ */
 export interface ISummaryTokenClaims {
```

Comment:
Hey @NicholasCouri you may have worked around protocol-defs and summaries. Could you comment on some of the interfaces here ? Unclear what is intended usage. I see token APIs/interfaces on driver layer, unclear on the protocol-defs use here.

Reply:
Hey @ssimic2 I'm not familiar with these interfaces - I wonder if @jatgarg  or @andre4i  would know :)

Reply:
`ISummaryTokenClaims` and `IActorClient` don't seem to be used anywhere. @GaryWilber do you know what these do?

Reply:
@GaryWilber in case you know the answer. I suppose that is the last info we need to proceed with the merge here.

Reply:
I'm not sure what those interfaces are. They are unused by ODSP/Push

Reply:
Should we mark these as deprecated then / remove them in next?

Reply:
Yes, I believe so!

Reply:
Marked as deprecated. Thanks!